dev-python/jaraco-context/jaraco-context-6.0.1.ebuild

```diff
@@ -23,7 +23,7 @@ KEYWORDS="~alpha amd64 arm arm64 hppa ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390
 RDEPEND="
 	$(python_gen_cond_dep '
 		dev-python/backports-tarfile[${PYTHON_USEDEP}]
-	' 3.10 3.11)
+	' 3.10 3.11 'pypy3*')
```

Comment:
There was an issue in `_python_impl_matches` function which was fixed in d133f3d0eb16 ("python-utils-r1.eclass: Fix matching pattern list starting with 3.10"). `3.11` is correctly matched even for `pypy` after the fix. However, proposed change would cause problems once pypy3.12 is introduced in the tree.